iamclient/iamclient.go

```diff
@@ -328,7 +336,8 @@ func (client *Client) getConnection(iam *remoteIAM) (*grpc.ClientConn, error) {
 }
 
 func (client *Client) sendIAMRequest(
-	nodeID string, requestFunc func(ctx context.Context, connection *grpc.ClientConn) error,
+	nodeID string, timeoutFunc func(*remoteIAM) time.Duration,
```

Comment:
Why do you need to pass function instead of just value?

Reply:
Let's do nil for default timeoutFunc.

Reply:
Done